docs/configuration/config-file-reference.md

```diff
@@ -2081,6 +2081,14 @@ alertmanager_client:
   # Skip validating server certificate.
   # CLI flag: -alertmanager.alertmanager-client.tls-insecure-skip-verify
   [tls_insecure_skip_verify: <boolean> | default = false]
+
+# The interval between persisting the current alertmanager state (notification
```

Comment:
> his state is read when all replicas for a shard have failed. 

This might be a little confusing - We'll read state from storage in the case of this replica being unable to communicate with the other e.g. transient network failures. Under that scenario, the other replicas are completely fine. 

Is it worth clarifying that it is not on actual replica failure but on the lack of a proper response from them?

Reply:
+1, I think it probably makes sense to say "when all replicas for a shard can not be contacted" instead, right, since won't know the cause, failure or network problem.

Reply:
Agreed, I'll update this.